src/lib/kit/hooks/useOneOf/TogglerCard/TogglerCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import {HelpPopover} from '@gravity-ui/components';
+import {Card, Text} from '@gravity-ui/uikit';
+
+import {block} from '../../../utils';
+
+import './TogglerCard.scss';
+
+const b = block('toggler-card');
+
+interface TogglerCardProps {
+    description?: string;
+    title: string;
+    text: string;
+    onOneOfChange: ([newValue]: string[]) => void;
+    disabled?: boolean;
+    value: string;
+    oneOfValue: string;
+}
+
+export const TogglerCard: React.FC<TogglerCardProps> = ({
```

Comment:
let's make this a more general component and move it to the components folder

Reply:
moved to components